src/mumble/IPCUtils.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021-2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_IPCUTILS_H_
+#define MUMBLE_MUMBLE_IPCUTILS_H_
+
+#include <QtCore/QDir>
```

Comment:
```suggestion
#include <QDir>
```
Including Qt stuff from "subdirectories" is discouraged - aka the wrong way to do it. (I know that lots of other Mumble code unfortunately still does it in the wrong way...)